resources/admin/classes/domain/Resource.php

```diff
@@ -1366,7 +1366,7 @@ public function getAlphabeticalList() {
 		$alphArray = array();
 		$result = $this->db->query("SELECT DISTINCT UPPER(SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)) letter, COUNT(SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)) letter_count
 								FROM Resource R
-								GROUP BY SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)
+								GROUP BY SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1), titleText
```

Comment:
I think we actually want the GROUP BY clause to match the first part of the SELECT clause. I'm guessing that would prevent the warning. I would suggest testing this:

GROUP BY UPPER(SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1))